.github/assets/kurtosis_op_network_params_local.yaml

```diff
@@ -9,10 +9,14 @@ optimism_package:
         el_image: "ghcr.io/ethereum-optimism/op-geth:kurtosis-ci"
         cl_type: op-node
         cl_image: "ghcr.io/ethereum-optimism/op-node:kurtosis-ci"
+        cl_extra_params:
+          - "--l1.trustrpc=true"
       - el_type: op-reth
         el_image: "ghcr.io/paradigmxyz/op-reth:kurtosis-ci"
         cl_type: op-node
         cl_image: "ghcr.io/ethereum-optimism/op-node:kurtosis-ci"
+        cl_extra_params:
+          - "--l1.trustrpc=true"
```

Comment:
this fixes op-node's problem right? nice

Reply:
yea